rumqttlog/src/logs/acks.rs

```diff
@@ -38,7 +38,7 @@ impl Acks {
         match qos {
             1 => self.acks.push(Packet::PubAck(PubAck::new(pkid))),
             2 => self.acks.push(Packet::PubRec(PubRec::new(pkid))),
-            _ => return,
+            _ => {}
```

Comment:
We might have missed a oxford `,` here?

Reply:
Commas after `{}` in match arms qre removed by rustfmt by default.